specification/eventgrid/data-plane/Microsoft.AppConfiguration/stable/2018-01-01/AppConfiguration.json

```diff
@@ -50,6 +50,58 @@
           "type": "string"
         }
       }
+    },
+    "AppConfigurationSnapshotCreatedEventData": {
```

Comment:
Yes, I added a base snapshot event properties. The name field is not in common list since we want to give different description about it. Please feel free to let know if you think they can be combined. There is only one minor difference on "created" or "modified".

Reply:
@JoshLove-msft After I synced up with team members, we decided to let each event to have its own properties separately.

Reply:
Based on offline discussion, we agree on solution of base event, and code has been updated